cmd/suggestion/pbt/v1beta1/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM python:3.9
```

Comment:
Can you add this Dockerfile to the [publish-algorithm-images.yaml](https://github.com/kubeflow/katib/blob/10f674a15512a2f1f2ebe52cab235d8702a2e1e4/.github/workflows/publish-algorithm-images.yaml)?